maestro-inspectors/pom.xml

```diff
@@ -30,6 +30,7 @@
 
     <modules>
         <module>maestro-activemq-inspectors</module>
+        <module>maestro-amqp-inspectors</module>
```

Comment:
I don't think this should be part of this PR ... at least, not right now. I believe you want to provide the infrastructure first and the implementation in a second PR, right?

Reply:
Correct, going to fix it.